README.md

```diff
@@ -1,17 +1,265 @@
+    public static IReviewConditionsBuilder<TReviewSettings> ApplicationOnboardingCompleted<TReviewSettings>(this IReviewConditionsBuilder<TReviewSettings> builder)
+        where TReviewSettings : ReviewSettingsCustom
+    {
+        builder.Conditions.Add(new SynchronousReviewCondition<TReviewSettings>(
+            (reviewSettings, currentDateTime) => reviewSettings.HasCompletedOnboarding is true)
+        );
+        return builder;
+    }
+}
+```
+
+It's possible to customize the review conditions used by the service by using `ReviewConditionsBuilder` and passing it to the `ReviewService` constructor or by injecting it as a transient when using dependency injection.
+
+``` cs
+using ReviewService;
+
+var reviewConditionsBuilder = ReviewConditionsBuilder.Empty<ReviewSettingsCustom>()
+    .ApplicationOnboardingCompleted()
+    .MinimumPrimaryActionsCompleted(3)
+    .MinimumApplicationLaunchCount(3)
+    .MinimumTimeElapsedSinceApplicationFirstLaunch(TimeSpan.FromDays(5));
+```
+
+### Track Review Settings
+
+To track your custom review settings, you can create extensions for `IReviewService` and be sure to make them generic so they are usable with custom review settings.
+
+``` cs
+using ReviewService;
+
+/// <summary>
+/// Extensions of <see cref="IReviewService{TReviewSettings}"/>.
+/// </summary>
+public static class ReviewServiceExtensions
+{
+    /// <summary>
+    /// Tracks that the application onboarding has been completed.
+    /// </summary>
+    /// <typeparam name="TReviewSettings">The type of the object that we use for tracking.</typeparam>
+    /// <param name="reviewService"><see cref="IReviewService{TReviewSettings}"/>.</param>
+    /// <param name="ct">The cancellation token.</param>
+    /// <returns><see cref="Task"/>.</returns>
+    public static async Task TrackOnboardingCompleted<TReviewSettings>(this IReviewService<TReviewSettings> reviewService, CancellationToken ct)
+        where TReviewSettings : ReviewSettingsCustom
+    {
+        await reviewService.UpdateReviewSettings(ct, reviewSettings =>
+        {
+            return reviewSettings with { HasCompletedOnboarding = reviewSettings.HasCompletedOnboarding };
+        });
+    }
+}
+```
+
+## Testing
+
+This is what you need to know before testing and debugging this service. Please note that this may change and you should always refer to the [Apple](https://developer.apple.com/documentation/storekit/skstorereviewcontroller/3566727-requestreview#4278434) and [Android](https://developer.android.com/guide/playcore/in-app-review/test) documentation for the most up-to-date information.
+
+### Android
+
+- You can't test this service while debugging the application, the prompt won't show up. To test it, you will need to use the [internal application sharing](https://play.google.com/console/about/internalappsharing/) or the internal testing feature in Google Play Console. See [this](https://developer.android.com/guide/playcore/in-app-review/test) for more details.
```

Comment:
I'm not super familiar with this Google Play feature, but I think you should also be able to test in-app reviews using an internal alpha/beta track. Not the most convenient way, but worth mentioning it too, perhaps? https://developer.android.com/guide/playcore/in-app-review/test#internal-test-track

Reply:
Yes why not! 👍